ui/sgd_dockwidget_base.py

```diff
@@ -63,6 +63,12 @@ def setupUi(self, sgdDockWidgetBase):
         self.horizontalLayout.addWidget(self.guiFileListStatus)
         spacerItem1 = QtWidgets.QSpacerItem(1, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
         self.horizontalLayout.addItem(spacerItem1)
+        self.guiStreamBtn = QtWidgets.QPushButton(self.dockWidgetContents)
```

Comment:
When changing the UI, you should do this via editing the *.ui file in the graphical editor, QtDesigner. This file is automatically generated and manual changes are lost when recreating it from the *.ui file.

The workflow is like this:
1. Open the ui file in QtDesigner, do your changes
2. Generate sgd_dockwidget_base.py by using command line tool pyuic5
`pyuic5 ui/sgd_dockwidget_base.ui -o ui/sgd_dockwidget_base.py`
3. Because pyuic5 has poblems with finding the resources file, you have to fix the last two lines of sgd_dockwidget_base.py:
```
from qgis import gui
from ..resources import resources
```